.github/workflows/build-test-deploy.yml

```diff
@@ -778,6 +778,25 @@ jobs:
           command: nix -- build .#price-feed
           token: "${{ secrets.CACHIX_AUTH_TOKEN }}"
 
+  package-dali-metadata:
```

Comment:
Should add this flow to effects-gate too

Reply:
added.